README.md

```diff
@@ -72,6 +72,7 @@ The community meeting calendar is available as an [iCal to subscribe to] (https:
   * Put a description of the topic in the welcome message
   * Set "Join the group" and "Post" to "Public"
 * Ask a repo maintainer to create a github label, if one doesn't already exist: area/foo
+* Ask organization maintainer to create a SIG-team within the [Kubernetes GitHub organization](https://github.com/orgs/kubernetes/teams)
```

Comment:
@sarahnovotny do we have a standard when we crate a SIG?  Since we have a ton of SIGs already, and we are waiting on more standards, I am wondering if it would be prudent to leave this off.